### PR TITLE
Videodevil fixes

### PR DESCRIPTION
**Pull request by Gujal00**

### CODE REVIEW

File: plugin.video.videodevil/videodevil.py [hunk 1]
```diff
@@ -655,6 +655,8 @@ def loadRemote(self, remote_url, recursive=True, lItem=None):
                 f = open(os.path.join(cacheDir, 'page.html'), 'w')
                 f.write('<Title>'+ curr_url + '</Title>\n\n')
             curr_url = urllib.unquote_plus(curr_url)
+            # Replace Space due to unqote_plus with + when using search terms with space
+            curr_url = curr_url.replace(' ','+')
```
Comment (mrdougquaid) [inline plugin.video.videodevil/videodevil.py]: Why not replace unquote_plus with unquote?

**Review by mrdougquaid — DISMISSED** [review]

File: plugin.video.videodevil/videodevil.py (hunk 1 above)
Comment (Gujal00) [reply plugin.video.videodevil/videodevil.py]: unquote_plus has always been there, didnt want to change and break any other stuff unintentionally.
So I added the line to replace spaces if any with + in the final url.

Comment (mrdougquaid) [reply plugin.video.videodevil/videodevil.py]: The difference between unquote and unquote_plus is that unquote_plus replaces '+' with ' ' . If you are undoing the extra bit done by unquote_plus you migh as well just use unquote.

Comment (Gujal00) [reply plugin.video.videodevil/videodevil.py]: The issue the unquote_plus is in two places as pointed out in #68, Don't know where it gets removed, So to be on the safe side I did it manually on the final url.
I can change the unquote_plus to an unquote and report back

Comment (Gujal00) [reply plugin.video.videodevil/videodevil.py]: I have tested with unquote on the second instance and it seems to work fine for both regualr urls and search urls with two words.
I have now made a commit with the change

**Comment by mrdougquaid** [comment]
This branch cannot be rebased due to conflicts
Rebasing the commits of this branch on top of the base branch cannot be performed automatically due to conflicts encountered while reapplying the individual commits from the head branch.

File: plugin.video.videodevil/resources/settings.xml [hunk 2]
```diff
@@ -23,6 +23,7 @@
     <setting id="Fux" type="bool" label="Fux" default="true" />
     <setting id="GirlfriendVideos" type="bool" label="GirlfriendVideos" default="true" />
     <setting id="Hentaigasm" type="bool" label="Hentaigasm" default="true" />
+	<setting id="JAVWhores" type="bool" label="JAVWhores" default="true" />
```
Comment (mrdougquaid) [inline plugin.video.videodevil/resources/settings.xml]: Use spaces not tabs please

File: plugin.video.videodevil/resources/settings.xml [hunk 3]
```diff
@@ -20,9 +20,9 @@
     <setting id="EPORNER" type="bool" label="EPORNER" default="true" />
     <setting id="ExtremeTube" type="bool" label="ExtremeTube" default="true" />
     <setting id="Fapdu" type="bool" label="Fapdu" default="true" />
-    <setting id="Fux" type="bool" label="Fux" default="true" />
     <setting id="GirlfriendVideos" type="bool" label="GirlfriendVideos" default="true" />
     <setting id="Hentaigasm" type="bool" label="Hentaigasm" default="true" />
+	<setting id="IndianMasala" type="bool" label="IndianMasalaSex" default="true" />
```
Comment (mrdougquaid) [inline plugin.video.videodevil/resources/settings.xml]: same here. Spaces not tabs otherwise it messes up the indentation

**Comment by mrdougquaid** [comment]
I cherry-picked the commits that had no issues. Thanks for your work.
Let me know when the rest is good to go.
Maybe better in the future to put each logical change in a separate pull request, that way all the easy ones get committed straight away

**Comment by imurreflection** [comment]
Thank you for releasing the partial update. Hope to get the full release soon

**Comment by Gujal00** [comment]
> 
> 
> I cherry-picked the commits that had no issues. Thanks for your work.
> Let me know when the rest is good to go.
> Maybe better in the future to put each logical change in a separate pull request, that way all the easy ones get committed straight away

I dont know how to create separate PRs, because once I open a PR any further commits get added to the same one and it wont let me create a new one until the first one is merged.
So for now I have adjusted the issues with settings.xml and committed. If it can't be merged, Iwill close this PR and Rebase my local git from your repo. Will create a new PR with all the other fixes when I have some time. Please advise

**Comment by mrdougquaid** [comment]
Separate PRs is for future requests. Have a look online. Essentially you create a new branch for each feature and send a PR for that branch

For this PR try updating your repo or something and that should get rid of the already committed commits

**Comment by Gujal00** [comment]
@mrdougquaid I think I have have cleaned the PR, please check and let me kow if it can be merged. Thanks for the info on using branches for PRs. Will follow in the future.

**Comment by mrdougquaid** [comment]
Still not right. 

See on this page "This branch cannot be rebased due to conflicts
Rebasing the commits of this branch on top of the base branch cannot be performed automatically due to conflicts encountered while reapplying the individual commits from the head branch."

Some of the already applied commits are still there.

**Comment by Gujal00** [comment]
Okay I will close this one, rebase and make a new PR

**Pull request closed without merging**